extensions/admin-tool-ext-win/package.json

```diff
@@ -75,5 +75,10 @@
 	},
 	"devDependencies": {
 		"vscode": "1.0.1"
+	},
+	"__metadata": {
+		"id": "41",
+		"publisherDisplayName": "Microsoft",
```

Comment:
Undo changes to this file

Reply:
Charles - I see these changes locally on build too. Is this a bug? Seems like these should be checked in (maybe in a separate PR)?

Reply:
Yeah I need to investigate what's causing these - they pop up randomly. Not sure if it's VSCode or something else adding it.